### PR TITLE
screen config passed to DrawerNavigatorItems components

### PR DESCRIPTION
**Pull request by ozanturksever**
I need to reach out to screen config to be able to customize drawer content. Like, to be able to decide hide/show some screen label in menu.

### CODE REVIEW

File: src/views/Drawer/DrawerNavigatorItems.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ type Props = {
   inactiveTintColor?: string;
   inactiveBackgroundColor?: string;
   getLabelText: (scene: DrawerScene) => string;
+  getScreenConfig: (routeKey: string, configName: string) => string;
```
Comment (satya164) [inline src/views/Drawer/DrawerNavigatorItems.js]: Let's pass `router` instead, and you can do `router.getScreenConfig(...)`

Comment (ozanturksever) [inline src/views/Drawer/DrawerNavigatorItems.js]: I am sorry,
you mean passing this.props.router? if so how can i call getScreenConfig, i don't have childNavigationProps at DrawerNavigationItems.

When i call router.getScreenConfig(navigation, 'label') i always get the first route configuration.

**Comment by grabbou** [comment]
@ozanturksever are you willing to re-base this PR and implement this feature? If not, let us know as we would like to land it soon.

**Comment by satya164** [comment]
Closing since over a month has passed without activity. Please send a new PR if you want to work on it.

**Pull request closed without merging**